docs/modules/manual/pages/beheerders.adoc

```diff
@@ -0,0 +1,106 @@
+= Authenticeer
+:description: Een gebruikershandleiding voor het configureren van beheerders van de erfgoedinstelling in Solid CRS.
+:sectanchors:
+:url-repo: https://github.com/netwerk-digitaal-erfgoed/solid-crs
+:imagesdir: ../images
+
+== Hoe log ik in als beheerder op het Collectiebeheersysteem van mijn erfgoedinstelling?
+
+Dit document toont je hoe je een beheerder kan configureren, zodat deze ook toegang heeft tot de Solid pod van de erfgoedinstelling. 
+
+
+=== Vereisten
+
+Om te kunnen inloggen is een WebID vereist. Heb je er nog geen, dan moet je eerst een link:url[WebID aanmaken]. 
+
+
+=== Verwijzen naar de erfgoedinstelling
+
+De beheerder dient in zijn eigen pod te verwijzen naar de catalogus van de erfgoedinstelling aan de hand van Type Registrations. In de Solid pod van de gebruiker, onder '/settings/publicTypeIndex.ttl' (of dergelijk), plaats je het volgende:
+
+[source]
+----
+@prefix solid: <http://www.w3.org/ns/solid/terms#>.
+@prefix schema: <http://schema.org/> .
+
+...
+
+<#hetlageland-catalog>
+  a solid:TypeRegistration;
+  solid:forClass schema:DataCatalog;
+  solid:instance <https://pods.nde.nl/hetlageland/heritage-collections/catalog>.
+----
+
+Waarbij `https://pods.nde.nl/hetlageland/` verwijst naar de pod van de erfgoedinstelling.
```

Comment:
Maybe be a little more clear that this is an example. Will the name of the institution only take up the last part of the url? If so, refer to this.